common/src/main/java/apoc/util/FileUtils.java

```diff
@@ -97,6 +101,13 @@ public static SupportedProtocols from(String source) {
                     // otherwise we return unknown protocol: yyyyy
                     return SupportedProtocols.valueOf(new URI(source).getScheme());
                 } catch (Exception ignored) {}
+
+                // in case a Windows user write an url like `C:/User/...`
+                if (e.getMessage().contains("unknown protocol") && Util.isWindows()) {
+                    throw new RuntimeException(e.getMessage() +
+                            "\n Please note that in case of a Windows absolute path, it have to be explicit by prepending file: or without drive, " +
+                            "\n e.g. `file:C:/my/path/file` or `/my/path/file`, instead of `C:/my/path/file`");
```

Comment:
```suggestion
                             "\n Please note that for Windows absolute paths they have to be explicit by prepending 'file': or supplied without the drive, " +
                            "\n e.g. `file:C:/my/path/file` or `/my/path/file`, instead of `C:/my/path/file`");
```

Have you managed to test that this works with different drives? (I don't have a Windows machine so can't check it)

Reply:
Yep, it works properly.
I changed the wording